isic/login/migrations/0005_profile_hash_id.py

```diff
@@ -0,0 +1,38 @@
+# Generated by Django 3.2.9 on 2021-12-08 04:10
+
+from django.conf import settings
+import django.core.validators
+from django.db import migrations, models
+from hashids import Hashids
+
+
+def gen_hashid(apps, schema_editor):
+    Profile = apps.get_model('login', 'Profile')  # noqa: N806
+    hasher = Hashids(
```

Comment:
Could this just import `get_hashid_hasher` so we don't have to define the parameters twice?